uwblueprint/website-bp-be
# Implement changes required for Heroku hosting

## Notion ticket link

[Heroku Postgres: Look into Hosting a Postgres DB on Heroku & Backend](https://www.notion.so/uwblueprintexecs/Heroku-Postgres-Look-into-Hosting-a-Postgres-DB-on-Heroku-Backend-b7a634d610654ea9ae24a7b804097b31)



## Implementation description
* Change migrations to work with sequelize from bash
* Use ssl for heroku connection to postgres db
* Use different config and env files



## Steps to test
1. Use command `heroku pg:psql postgresql-solid-50144 --app sleepy-caverns-02101` to connect to the database and check it out
2. Make a graphql request like the following: 
```graphql
query {
    dashboardById(id: 2) {
        reviewerEmail
    }
}
```
at `https://sleepy-caverns-02101-edcc7706ba5e.herokuapp.com/graphql` (link might be outdated). 



## What should reviewers focus on?
* Currently firebase authentication doesn't quite work for this, as seen in certain endpoints (such as most of the user queries)


## Checklist
